pyHS100/pyHS100.py

```diff
@@ -291,8 +291,22 @@ def location(self):
         :rtype: dict
         """
         info = self.sys_info
-        return {"latitude": info["latitude"],
-                "longitude": info["longitude"]}
+        loc = {"latitude": 0.0,
+               "longitude": 0.0}
+
+        if "latitude" in info:
+            loc["latitude"] = info["latitude"]
+
```

Comment:
I would do this like following:
```
if "latitude" in info and "longitude" in info:
    # set both here
elif "latitude_i" in info and "longitude_i" in info:
    # the same
else:
    # log a warning about probably unsupported device?
```

It would also make sense to initialize latitude and longitude to None instead of 0.0, to differentiate between non-existing and well, 0.0/0.0 coordinate.